app/views/parent/kids/edit.html.erb

```diff
@@ -1,6 +1,6 @@
-<h1>Create a Kid!</h1>
+<h1> Edit Kid Information </h1>
 
-<%= form_for @kid do |form| %>
+<%= form_for @kid, url: "/kids/#{params[:id]}" do |form| %>
```

Comment:
Yeah I remember this; we were able to change which kid we were editing straight through the inspect page. There's a way of implementing a `_path` prefix to it (like most other pages) however we'd still need to feed the ID into it.